src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig

```diff
@@ -32,34 +32,48 @@
 
 
 {% block content %}
-  <div class="row">
-    <div class="col-md-12">
-      {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig' %}
+  <div id="order-view-page">
```

Comment:
Be careful with these modifications as they will conflict with https://github.com/PrestaShop/PrestaShop/pull/16144 if it is migrated sooner
This bring the question about how content from hooks will be interpreted during print.. Although you can't guess what they will print
Same question with the tabs from hooks https://github.com/PrestaShop/PrestaShop/pull/16144/files#diff-1a42492ba7a3b1fc9477d3ad77b8edb5R70
Which probably won't integrate card to optimize print display

Reply:
Well it will show the content of every tabs as it is doing with payment, shipping etc.. Should we force the display of default tabs and only these ?

Reply:
But I guess the card are used in order to have the title right before the content?
If so then I guess the tabs from hooks will look like:
```
Title 1
Title 2
Content 1
Content 2
```